lib/web_ui/dev/test_runner.dart

```diff
@@ -351,6 +351,9 @@ class TestCommand extends Command<bool> with ArgUtils {
   /// Whether [browser] is set to "safari".
   bool get isSafariOnMacOS => browser == 'safari' && io.Platform.isMacOS;
 
+  /// Whether [browser] is set to "ios-safari".
+  bool get isSafariIOS => browser == 'ios-safari' && io.Platform.isMacOS;
```

Comment:
Should the second part be `io.Platform.isIOS`?

Reply:
Very good question. Inside the tests we are doing the check as iOS (since tests runs on an iOS simulator), but this is the test_runner which runs on the LUCI bot, which would be a macOS machine.